cmd/loki/loki-local-config.yaml

```diff
@@ -5,7 +5,7 @@ server:
   grpc_listen_port: 9096
 
 common:
-  path_prefix: /tmp/loki
+  path_prefix: /var/lib/loki
```

Comment:
If you're changing this value, you'll also need to change the `chunks_directory` and `rules_directory` values.

Have you validated that this works?

Reply:
Thanks for the feedback @dannykopping.
May you please clarify me which `chunks_directory` and `rules_directory ` are you refrencing to ?

Reply:
Look a few lines down in the config file

Reply:
so I've to change `/tmp/loki/chunks` to `var/lib/loki/chunks` and same for the rules directory?

Reply:
Yes @KanishkaVerma054 , exactly :)